src/main_mod_sup.erl

```diff
@@ -0,0 +1,45 @@
+
+-module(main_mod_sup).
```

Comment:
and here too with the namespace thing 😄

Reply:
OK, prefixed files and code reference with xqerl